spec/system/dashboard/callouts_spec.rb

```diff
@@ -0,0 +1,191 @@
+require 'rails_helper'
+
+RSpec.describe 'Callout pages', type: :system do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in(user)
+  end
+
+  describe 'view all callout' do
+    it 'should list all callouts of current account' do
+      account_callout = create(:callout, account: user.account, metadata: { title: 'My Callout'})
+      other_callout = create(:callout, metadata: { title: 'Other Callout'})
+
+      visit dashboard_callouts_path
+
+      expect(page).to have_text('My Callout')
+      expect(page).not_to have_text('Other Callout')
```

Comment:
this one you even can create a helper method

`expect(page).to have_callout(my_callout)`

```ruby
def have_callout(callout)
  have_selector(...) # base on css selector
end
```